### PR TITLE
SDIT-2044 A few more fields to PERSON - language

### PR DESCRIPTION
**Pull request by andymarke**
Fixed a couple of warnings

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Person.kt [hunk 1]
```diff
@@ -78,9 +77,30 @@ data class Person(
...
+  CRIMINAL_HISTORY_TEXT - always null
+  NAME_TYPE - always null
+  ALIAS_PERSON_ID - always null
+  ROOT_PERSON_ID - always null
+
+
+   */
+
 ) {
-  fun getEmails(): List<PersonInternetAddress> = internetAddresses.stream().filter { ia: PersonInternetAddress -> "EMAIL" == ia.internetAddressClass }
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Person.kt:82]: not used - deleted

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/NomisData.kt [hunk 2]
```diff
@@ -27,7 +27,6 @@ class NomisDataBuilder(
   private val nonAssociationBuilderFactory: NonAssociationBuilderFactory? = null,
   private val externalServiceBuilderFactory: ExternalServiceBuilderFactory? = null,
   private val agencyInternalLocationBuilderFactory: AgencyInternalLocationBuilderFactory? = null,
-  private val courtCaseBuilderFactory: CourtCaseBuilderFactory? = null,
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/NomisData.kt:30]: not used - deleted

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/NomisData.kt [hunk 3]
```diff
@@ -165,7 +162,7 @@ class NomisData(
   override fun incident(
     title: String,
     description: String,
-    agencyId: String,
+    locationId: String,
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/NomisData.kt:165]: name different to interface warning

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**